test-website/bad-encoding.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Test website</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="./icons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./icons/favicon-16x16.png">
+    <link rel="manifest" href="./icons/site.webmanifest">
+    <link rel="shortcut icon" href="./icons/favicon.ico">
+    <link rel="stylesheet" href="./css-broken-file.css">
+</head>
+
+<body>
+
+    <h2>Bad encoding</h2>
+
+    <p>
+        <img src="http://www.mediaelementjs.com/images/big_buck_bunny.jpg">
+    </p>
+
+    <p>Below is some bad encoding. This file has been saved with Windows 1252 and content below comes from Simplified Chinese GB2312. Typically happens when content is merged from various sources, e.g. in dynamic websites.</p>
```

Comment:
Maybe add in the description that html meta declare encoding as utf-8 (and that firefox decode this content as utf-8 "by default")

Reply:
I mentioned the html meta but not the firefox thing, this is too specific and subject to change